sqlmesh/dbt/manifest.py

```diff
@@ -398,6 +399,9 @@ def _extra_dependencies(self, target: str, package: str) -> Dependencies:
         for call_name, node in extract_call_names(target, cache=self._calls):
             if call_name[0] == "config":
                 continue
+            elif isinstance(node, jinja2.nodes.Getattr):
+                if call_name[0] == "model":
+                    dependencies.macros.append(MacroReference(name=".".join(call_name)))
```

Comment:
Repurposing macro references for this feels like a hack. Can we have a separate set in `Dependencies` instead?